includes/tracks/class-wc-tracks-client.php

```diff
@@ -110,7 +148,7 @@ public static function get_anon_id() {
 					$binary .= chr( wp_rand( 0, 255 ) );
 				}
 
-				$anon_id = 'jetpack:' . base64_encode( $binary );
+				$anon_id = 'woo:' . base64_encode( $binary );
 
 				if ( ! headers_sent()
```

Comment:
>Could use `wc_setcookie` helper here. If those other 2 checks are worth (?) you could improve `wc_setcookie` in the process.

~ https://github.com/woocommerce/woocommerce/pull/22625#discussion_r257179911

Reply:
>This probably needs some sort of sanitization before returning it because it's saved straight to meta.

in reference to `$anonid` in https://github.com/woocommerce/woocommerce/pull/22625/files#r257180446